beancount/scripts/doctor.py

```diff
@@ -7,67 +7,119 @@
 __license__ = "GNU GPLv2"
 
 import collections
+import logging
 import os
 import re
 import sys
-import argparse
-import logging
-from os import path
+
+import click
 
 from beancount import loader
 from beancount.core import account
 from beancount.core import account_types
 from beancount.core import compare
 from beancount.core import convert
 from beancount.core import data
-from beancount.core import display_context
+from beancount.core.display_context import Align
 from beancount.core import getters
 from beancount.core import inventory
 from beancount.core import prices
 from beancount.core import realization
-from beancount.parser import context
+from beancount.parser.context import render_file_context
```

Comment:
It's an artifact of using Google style for so many years that I tend to favor always importing packages over functions directly.

Reply:
I would not have changed it if there would not have been a collision with a function with the same name. I though that the easier solution was to reduce the import scope.